float64 timestamp
float64[6] target_q
float64[6] target_qd
float64[6] target_qdd
float64[6] target_current
float64[6] target_moment
float64[6] actual_q
float64[6] actual_qd
float64[6] actual_current
float64[6] joint_control_output
float64[6] actual_TCP_pose
float64[6] actual_TCP_speed
float64[6] actual_TCP_force
float64[6] target_TCP_pose
float64[6] target_TCP_speed
uint64 actual_digital_input_bits
float64[6] joint_temperatures
float64 actual_execution_time
int32 robot_mode
int32[6] joint_mode
int32 safety_mode
float64[3] actual_tool_accelerometer
float64 speed_scaling
float64 target_speed_fraction
float64 actual_momentum
float64 actual_main_voltage
float64 actual_robot_voltage
float64 actual_robot_current
float64[6] actual_joint_voltage
uint64 actual_digital_output_bits
uint32 runtime_state
float64[3] elbow_position
float64[3] elbow_velocity
uint32 robot_status_bits
uint32 safety_status_bits
uint32 analog_io_types
float64 standard_analog_input0
float64 standard_analog_input1
float64 standard_analog_output0
float64 standard_analog_output1
float64 io_current
uint32 euromap67_input_bits
uint32 euromap67_output_bits
float64 euromap67_24V_voltage
float64 euromap67_24V_current
uint32 tool_mode
uint32 tool_analog_input_types
float64 tool_analog_input0
float64 tool_analog_input1
int32 tool_output_voltage
float64 tool_output_current
float64 tool_temperature
uint32 output_bit_registers0_to_31
uint32 output_bit_registers32_to_63
bool[64] output_bit_register
int32[48] output_int_register
float64[48] output_double_register
uint32 input_bit_registers0_to_31
uint32 input_bit_registers32_to_63
bool[64] input_bit_register
int32[48] input_int_register
float64[48] input_double_register
